src/Command/Pull/PullDatabaseCommand.php

```diff
@@ -19,7 +19,8 @@ class PullDatabaseCommand extends PullCommandBase {
    * {inheritdoc}.
    */
   protected function configure() {
-    $this->setDescription('Copy database from a Cloud Platform environment')
+    $this->setDescription('Import latest database backup from a Cloud Platform environment')
+      ->setHelp('This uses the latest available database backup, which may be up to 24 hours old. You can generate an on-demand backup using api:environments:database-backup-create.')
```

Comment:
As a user, when am I supposed to read this message?

Reply:
You'll see the help if you run `acli help pull:db`. I debated also making it part of the command output (when pulling the db) but that seemed too verbose.